smithery.yaml

```diff
@@ -1,25 +1,7 @@
 version: 1
-start:
-  command: ["node", "dist/index.js"]
-  env:
-    - name: NODE_ENV
-      value: "production"
-
+runtime: "container"
```

Comment:
And you reviewd the Dockerfile? If the deployment fails in Smithery, would users access the old deployment?